addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -34,6 +35,29 @@ export class MassMailingHtmlField extends HtmlField {
         this.orm = useService('orm');
         this.dialog = useService('dialog');
 
+        const onIframeUpdated = this.onIframeUpdated;
+        this.onIframeUpdated = () => {
+            onIframeUpdated();
+            this._updateIframe();
+        }
+        const throttledOnResizeObserved = useThrottleForAnimation(() => {
+            this._resizeMailingEditorIframe();
+            this._repositionMailingEditorSidebar();
+        });
+        this._resizeObserver = new ResizeObserver(throttledOnResizeObserved);
+        const throttledRepositionSidebar = useThrottleForAnimation(
+            this._repositionMailingEditorSidebar.bind(this)
+        );
+        onMounted(() => {
+            $('.o_content').on('scroll.repositionMailingEditorSidebar', throttledRepositionSidebar);
+            $('.o_form_sheet_bg').on('scroll.repositionMailingEditorSidebar', throttledRepositionSidebar);
+        });
+        onWillUnmount(() => {
+            $('.o_content').off('.repositionMailingEditorSidebar');
+            $('.o_form_sheet_bg').off('.repositionMailingEditorSidebar');
```

Comment:
I believe here, the concept you went with is: for every known "scrollable", listen to the scroll event, then re-dimension, right ?

Can't we add  a listener to the scroll event on `document` only, and check if the `event.target` contains the iframe instead (to know when to execute the callback) ? (this will be more robust in case more scrollable are added, or if their classes change)

You can use `useExternalListener` which will handle `onMounted and onWillUnmount` and binding the function